core/src/test/java/io/questdb/cutlass/line/tcp/LineTcpReceiverTest.java

```diff
@@ -349,8 +348,9 @@ public int openRW(LPSZ name, long opts) {
                     weather + ",location=us-eastcoast temperature=81 1465839830101400200\n" +
                     weather + ",location=us-midwest,source=sensor1 temp=85 1465839830102300200\n" +
                     weather + ",location=us-eastcoast,source=sensor2 temp=89 1465839830102400200\n" +
-                    weather + ",location=us-westcost,source=sensor1 temp=82 1465839830102500200\n";
-            send(receiver, lineData, weather, WAIT_ILP_TABLE_RELEASE);
+                    weather + ",location=us-westcost,source=sensor1 temp=82 1465839830102500200\n" +
+                    "done ok=t\n";
```

Comment:
what is the reason for creating the done table?
is it not enough to wait for the weather ILP release event?

Reply:
`weather` WalWriter will be released 2 times because in this test table `weather` is created twice.
It's created first, written to, then dropped and released.
Then it's re-created written again and released. 
There is no test api to wait for a table to be released twice, simplest is to add dummy table and wait for the release of it.

Reply:
makes sense now, thanks